utils/prompt.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// confirm prompts a user for a confirmation with message
+// and returns true with no error if input is "yes" or "y" (case-insensitive),
+// otherwise false.
+func Confirm(message string) (bool, error) {
+	fmt.Print(message)
+	yes := []string{"yes", "y"}
+	var input string
+	_, err := fmt.Scanln(&input)
+	if err != nil {
+		return false, err
+	}
+	input = strings.ToLower(input)
+	for _, valid := range yes {
+		if input == valid {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func ConfirmFileOverwrite(filename string, ext string, assumeYes bool) (bool, error) {
```

Comment:
lint: public symbol needs godoc comment

```suggestion
// ConfirmFileOverwrite is a helper function to determine whether or not the program should
// truncate and overwrite a file given its name and extension. If the file doesn't already exist
// in the filesystem, then this will return true, otherwise it will prompt the user for confirmation.
func ConfirmFileOverwrite(filename string, ext string, assumeYes bool) (bool, error) {
```